.doxygen

```diff
@@ -160,8 +160,9 @@ FILE_PATTERNS          = *.c \
                          *.ucf \
                          *.qsf
 RECURSIVE              = YES
-EXCLUDE                = mwg build src/python src/third_party \
-                         src/gennylib/test src/driver/test src/cast_core/test
+EXCLUDE                = build src/python src/third_party \
```

Comment:
I noticed this before, but I'm not certain `EXCLUDE` works for directories. Perhaps [EXCLUDE_PATTERNS](https://stackoverflow.com/questions/5507055/excluding-directories-for-doxygen)

Reply:
I tried a few times with `EXCLUDE_PATTERNS` and it always ends up grabbing `third_party` etc.  FWIW `EXCLUDE` does seem to work as-is here